### PR TITLE
Hubert / implementing branding removal component

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: src/theme/Navbar/Logo/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Logo from '@theme/Logo';
+import RenderOfficialContents from '@site/src/components/RenderOfficialContents';
+export default function NavbarLogo() {
```
Comment (matin-deriv) [inline src/theme/Navbar/Logo/index.js:4]:
```suggestion
export const NavbarLogo = () => (
<Render...
```

File: src/utils/index.ts [hunk 2]
```diff
@@ -24,6 +24,19 @@ const CURRENCY_MAP = new Map([
   ['AUD', { icon: 'aud', name: 'Australian Dollar' }],
 ]);
 
+export const domains = [
+  'deriv.com',
```
Comment (matin-deriv) [inline src/utils/index.ts:28]: Do you need to include deriv.be and deriv.me as well?

File: src/theme/Navbar/Content/index.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+import { splitNavbarItems, useNavbarMobileSidebar } from '@docusaurus/theme-common/internal';
+import NavbarItem from '@theme/NavbarItem';
+import NavbarColorModeToggle from '@theme/Navbar/ColorModeToggle';
+import SearchBar from '@theme/SearchBar';
+import NavbarMobileSidebarToggle from '@theme/Navbar/MobileSidebar/Toggle';
+import NavbarLogo from '@theme/Navbar/Logo';
+import NavbarSearch from '@theme/Navbar/Search';
+import useOfficialContentsContext from '@site/src/hooks/useOfficialContentsContext';
+import styles from './styles.module.css';
+function useNavbarItems() {
```
Comment (matin-deriv) [inline src/theme/Navbar/Content/index.js:12]: Is it possible to update to ES6 functions?
we have multiple functions in this file.

File: src/theme/Navbar/ColorModeToggle/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useColorMode, useThemeConfig } from '@docusaurus/theme-common';
+import ColorModeToggle from '@theme/ColorModeToggle';
+import styles from './styles.module.css';
+export default function NavbarColorModeToggle({ className }) {
```
Comment (matin-deriv) [inline src/theme/Navbar/ColorModeToggle/index.js:5]: ditto

File: src/theme/DocPaginator/index.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Translate, { translate } from '@docusaurus/Translate';
+import PaginatorNavLink from '@theme/PaginatorNavLink';
+import RenderOfficialContents from '@site/src/components/RenderOfficialContents';
+export default function DocPaginator(props) {
```
Comment (matin-deriv) [inline src/theme/DocPaginator/index.js:5]: ditto

**Review by matin-deriv — DISMISSED** [review]

File: src/theme/Navbar/Logo/index.js (hunk 1 above)
Comment (hubert-deriv) [reply src/theme/Navbar/Logo/index.js:4]: The components in the theme folder are third party components. Sometimes we need to swizzle (extract) them and only for some necessary adjustments. I think changing the export style might affect other node packages/components that import these swizzled components.

File: src/utils/index.ts (hunk 2 above)
Comment (hubert-deriv) [reply src/utils/index.ts:28]: added

**Pull request merged**